CHANGELOG.md

```diff
@@ -1,3 +1,143 @@
+* [Adding Snapshot tests for the various Chip States (#7269)](https://github.com/material-components/material-components-ios/commit/e5aa9c946509ab9b8cea20fcf306256ba6132f6c) (Yarden Eitan)
+* [add missing theming extension documentation. (#7194)](https://github.com/material-components/material-components-ios/commit/47cef3068a0a8d7a9625584b170bd373d04b94dd) (Wenyu Zhang)
+
+### Dialogs
+
+* [Graduate Dialogs theming extensions to ready (#7190)](https://github.com/material-components/material-components-ios/commit/bf5d5e1c1df23cb73aea88f22e465480cf875f00) (Andrew Overton)
+
+### Ripple
+
+* [Add better resizing animation support (#7271)](https://github.com/material-components/material-components-ios/commit/fab77eef78a0ba6aefbdaff38aa66d3446e8ffc7) (Yarden Eitan)
+* [Additional snapshot test for an edge case (#7279)](https://github.com/material-components/material-components-ios/commit/a8a9cfa534f0fbfb8051fc85046c40695138884d) (Yarden Eitan)
+* [Don't dissolve the ripple if the highlighted state is still set to YES. (#7274)](https://github.com/material-components/material-components-ios/commit/d41b6530d60dad78a4d33d57c186a76e8f44149e) (Yarden Eitan)
+* [Minor docs clean up (#7262)](https://github.com/material-components/material-components-ios/commit/bd127b3ea934379e6d054e5646db4274ea624b89) (Cody Weaver)
+* [Reduce runloop time as speed is already very fast (#7283)](https://github.com/material-components/material-components-ios/commit/2cfe2057501411279a358481f73bfd6986cb76a1) (Yarden Eitan)
+
+### TextFields
+
+* [Correct UIAccessibility values and behavior. (#7256)](https://github.com/material-components/material-components-ios/commit/4ffa929e881d6ba09cd9bca0956136d3b4c42969) (Robert Moore)
+* [Fix Contained example error states. (#7252)](https://github.com/material-components/material-components-ios/commit/62cca706aea866a03658032e1f514691d04c99f2) (Robert Moore)
+* [Fix character count overflow. (#7255)](https://github.com/material-components/material-components-ios/commit/63044ff567278d52e452b194617c78035a9578a7) (Robert Moore)
+* [Fix example layout. (#7251)](https://github.com/material-components/material-components-ios/commit/b59abd2dd7e0fc2a6f46d401c7a8529330bfbafb) (Robert Moore)
+* [Fix examples layout (#7250)](https://github.com/material-components/material-components-ios/commit/0c317f190d07259144ccd475018dca5bf4bd5232) (Robert Moore)
+* [Make Kitchen Sink better for tests. (#7259)](https://github.com/material-components/material-components-ios/commit/97ebc0ce0bd5ea48a9e71830c6b342bb6cccb78c) (Robert Moore)
+
+## Multi-component changes
+
+* [ [Catalog] Add @objc annotations to our containerScheme instances in Swift (#7243)](https://github.com/material-components/material-components-ios/commit/52da482fd2665dfc66760f0f0880baa23ac16384) (Yarden Eitan)
+* [update (#7246)](https://github.com/material-components/material-components-ios/commit/efe61588a601606d3cdc48652f8d63c662b4fc9d) (Yarden Eitan)
```

Comment:
Should this be marked as `[Catalog]` like the thing above it? They are both about adding `@objc` annotations to properties in examples

Reply:
These changes are auto-generated, so fixes to this will need to be captured as bugs and then fixes applied to the release tool.

Reply:
Made the fix but I can't find the repo to add a bug to

Reply:
actually upon further examination the commit was not amended properly to use the PR's description. This is not something the API diff too can fix. Unless we want it to ignore the Commit message and use the resolved PR description?

Reply:
filed https://github.com/material-motion/apidiff/issues/28 anyway.